src/lucky/tasks/gen/templates/resource/pages/{{folder_name}}/index_page.cr.ecr

```diff
@@ -4,7 +4,7 @@ class <%= pluralized_name %>::IndexPage < MainLayout
 
   def content
     h1 "All <%= pluralized_name %>"
-    link "New <%= resource %>", to: <%= pluralized_name %>::New
+    link "New <%= resource %>", <%= pluralized_name %>::New
```

Comment:
I personally prefer there to be the `to:`. I always add it in my projects. But since line 15 is defined without it, I guess we can do this. Or we need to update line 15 to include the `to:` just so they both match.